nixos/modules/config/pulseaudio.nix

```diff
@@ -135,6 +177,7 @@ in {
 
     (mkIf cfg.enable {
       environment.systemPackages = [ cfg.package ];
+      environment.sessionVariables = moduleEnvVars;
```

Comment:
Sounds like a good idea :) I'll implement it along with your other suggestions when I can then redo my commit.